manifest_merging/manifest_merging_job.py

```diff
@@ -0,0 +1,75 @@
+"""
+Module for bucket manifest merging using sower-jobs
+"""
+
+import os
+import sys
+import json
+import logging
+import boto3
+
+from gen3.tools.indexing import merge_bucket_manifests
+
+from settings import JOB_REQUIRES
+from utils import (
+    check_user_permission,
+    detect_delimiter,
+    split_s3_path,
+    upload_file_to_s3_and_generate_presigned_url,
+)
+
+LOG_FILE = "/manifest_merging.log"
+INPUT_MANIFESTS_DIRECTORY = "/input_manifests"
+OUTPUT_MANIFEST = "/output_manifest.txt"
```

Comment:
I went with `.txt` as a general extension to cover all possible delimiters. I figured that it doesn't matter too much since this job supplies `merge_bucket_manifests` with `output_manifest_file_delimiter` (which is set based on the delimiter used in the last input manifest downloaded). Also, with the current implementation of `upload_file_to_s3_and_generate_presigned_url`, `OUTPUT_MANIFEST` won't have an effect on the file extension of the output S3 presigned url.